docs/users.md

```diff
@@ -314,6 +316,12 @@ To create a control plane of type `host` run the command:
 kflex create cp4 --type host
 ```
 
+To create a control plane of type `external` with the required options, run the command: 
```

Comment:
This command will also create a Secret object that lingers forever, right? If so then that seems like something worth mentioning. And if the Secret is eventually automatically deleted, users should be told about its tempoary existence.